brave/brave-ios
# Payment Request API support (BAP only)

Fixes https://github.com/brave/brave-ios/issues/2386

### Introduction
This PR adds support to use BAT as a payment instrument for the Payment Request API. 

### Test Plan

1. Enable rewards
2. Navigate to https://alexwykoff.com/store/brave/store.php
3. Click on Buy 
4. Should show the Payment UI
5. When buy is clicked 2 BAT should be deducted from the wallet balance.

### To-Do:

- [x] if BAT is not in the list of supported instruments, reject the response.
- [x] Support only for secure contexts
